### DISCUSSION TITLE
Migration strategy for 2.0 regarding backups

### DISCUSSION THREAD

**Discussion by drewbitt** [lnreader/lnreader#1126] [General]
> The backup data from version 1.1.19 does not include downloads or reading progress information. As a result, any downloaded content and your progress in chapters will be lost and not carried over when upgrading to version 2.0.0.

Can we not add reading progress to v1 backups in another release before pivoting to v2? I just attempted the migration and it was not pleasant losing everything.

**Comment by ovizii** [comment]
I have the same question, v 1.1.19 started crashing, I don't even know where I can find all the books I had downloaded to migrate, not all that concerned about reading progress though. A migration strategy would be great.

**Comment by drewbitt** [comment]
Unfortunately this was not considered before 2.0. best of luck everyone manually migrating

**Reply by ovizii** [reply]
Check this, although I am not sure what exactly it can convert: https://www.lnreader.app/tools/backup-upgrader progress or not!?
